uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/leave-request-model_test.js

```diff
@@ -10,12 +11,14 @@ define([
       LeaveRequestAPI,
       $rootScope;
 
-    beforeEach(module('leave-absences.models', 'leave-absences.mocks', function (_$provide_) {
-      $provide = _$provide_;
+    beforeEach(module('leave-absences.models', 'leave-absences.mocks', 'common.mocks',
+      function (_$provide_) {
+        $provide = _$provide_;
     }));
 
-    beforeEach(inject(function (_LeaveRequestAPIMock_) {
+    beforeEach(inject(function (_LeaveRequestAPIMock_, _FileUploaderMock_) {
       $provide.value('LeaveRequestAPI', _LeaveRequestAPIMock_);
+      $provide.value('FileUploader', _FileUploaderMock_);
```

Comment:
Is it necessary to mock `FileUploader` in each file? I commented line 21 of this file and test cases ran successfully. 
Please check all files where mock has been added.